recipes/cross-python/meta.yaml

```diff
@@ -0,0 +1,27 @@
+{% set build_number = "0" %}
+package:
+  name: cross-python
+  version: {{ version }}
+
+build:
+  number: {{ build_number }}
+  string: {{ build_number }}_cpython
+  skip: True  # [win]
+
+requirements:
+  run:
+    - python {{ version }}.* *_cpython
+    - crossenv
```

Comment:
Shouldn't this have an rysnc dep for the activation script?

Reply:
Yes, it should